src/ConfigurationProcessor.php

```diff
@@ -14,8 +14,21 @@
  */
 final class ConfigurationProcessor
 {
+    /**
+     * @var string the modules path
+     */
     const MODULES_PATH = '/modules/';
 
+    /**
+     * @var int the number of allowed parallel PHP processes
+     */
+    const PARALLEL_PROCESSES = 8;
```

Comment:
**Warning** no matter what, this is not in the scope of this pull request ;)

Reply:
@mickaelandrieu If you add something in the code, this is in the scope of the pull request..

Reply:
You don't need to calculate the number of processes, however you should use good naming for this const since it's introduced by this PR
As it's the default value (and you can override it), `DEFAULT_PARALLEL_PROCESSES` is more appropriate. Of course the same feedback is valuable for other const I didn't want to multiple the same comments but you should rename them as well... please ^^

Reply:
Alright I didn't see you changed them. My bad.